packages/terra-clinical-item-view/src/ItemView.module.scss

```diff
@@ -102,6 +102,10 @@
       flex: 1 1 auto;
       float: left;
     }
+
+    .content {
+      min-height: 1.25em; // Setting a line height so when a blank display is passed in the content maintains it's height instead of shrinking
+    }
```

Comment:
Ideally I think we would want to apply this min height only when the display is empty. However I wasn't able to figure out how to do that. I tried some variations of the below, but I must be missing something. If needed I think it should be fine to not set min height on just the empty displays, this 1.25em value doesn't mess with any of the other snapshots.
    
```
.content:blank {
    min-height: 1.25em;
}
```
```
.content:empty {
   &::before {
      min-height: 1.25em;
   }
}
```

Reply:
I am curious if we are able to set it on the implementation side. Maybe adding the display empty check here https://github.com/cerner/terra-clinical/blob/blank-display-height-fix/packages/terra-clinical-item-view/src/ItemView.jsx#L210-L212?